languages/messages/wikia/crowdin.conf

```diff
@@ -0,0 +1,4 @@
+[crowdin.project]
+projectid = "CoreMessagesWikia"
+format = "mediawiki"
+source.file = "MessagesEn.php"
```

Comment:
Yeah :( though when I've done this for these files before, I've just created a temporary i18n.php file locally and used it to do the upload and later download, then just copied the translations to the right files in languages/messages/wikia. Ugly, but at least you get the conversion to and from JSON.